tests/postgres_tests/test_indexes.py

```diff
@@ -98,12 +104,12 @@ def test_gin_index(self):
 
     @skipUnlessDBFeature('has_brin_index_support')
     def test_brin_index(self):
-        index_name = 'char_field_model_field_brin'
+        index_name = 'char_field_model_field_brn'
         index = BrinIndex(fields=['field'], name=index_name, pages_per_range=4)
         with connection.schema_editor() as editor:
             editor.add_index(CharFieldModel, index)
         constraints = self.get_constraints(CharFieldModel._meta.db_table)
-        self.assertEqual(constraints[index_name]['type'], BrinIndex.suffix)
+        self.assertEqual(constraints[index_name]['type'], 'brin')
```

Comment:
This goes against the ideas proposed in [#27135](https://code.djangoproject.com/ticket/27135). If we change the suffix, we need a mapping somewhere so that introspection continues to return `BrinIndex.suffix`.